src/extension.ts

```diff
@@ -40,6 +58,12 @@ export function activate(context: vscode.ExtensionContext) {
 
 		}).catch((err:any) => {
 			console.log("ERROR: " + err);
+			if(err === ECONNREFUSED) {
+				vscode.window.showErrorMessage('Kindly check the URL you have provided and ensure firewall has been disabled');
+			}
+			if(err.status === 401) {
```

Comment:
Still learning my way around typescript, but on googling `err.status` seems to be the logical choice to catch an unauthorized status.